### PR TITLE
[11.0][stock_quantity_history_location]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
The modules added here allow to run an Inventory report or Inventory Valuation report by location, for a past date or for current date.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@jbeficent @hveficent Is this WIP or needs review ?

**Comment by HviorForgeFlow** [comment]
@rousseldenis I don't think so, we are currectly using it, a review would be great. :smile:
Might tests are needed

**Comment by rousseldenis** [comment]
@jbeficent @hveficent If I understand, the module 'stock_account_quantity_history_location' usage is to warn user that accounting valuation is not computed when specifying location?

**Comment by HviorForgeFlow** [comment]
@rousseldenis yes, at least, this is the purpose.

File: stock_account_quantity_history_location/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Glue module between Stock Account and Stock Quantity History Location modules.
+
+Accounting valuation is not calculated when you specify a location.
```
Comment (rousseldenis) [inline stock_account_quantity_history_location/readme/DESCRIPTION.rst]:
```suggestion
Display a warning on the Stock Inventory report that accounting valuation is not calculated when you specify a location.
```

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM code review

**Pull request closed without merging**